### DISCUSSION TITLE
Lift 4.0.0-M1

### DISCUSSION THREAD

**Discussion by farmdawgnation** [lift/framework#2021] [Announcements]
The Lift Committers are pleased to announce the release of Lift 4.0.0-M1 on September 2nd, 2024. This is the first release of the Lift 4.0 series, which will be a stepping-stone release to Lift 5.0 which will include a host of breaking changes. This release does include breaking changes.

## Breaking Changes

Per ongoing conversations around slimming down the Lift Framework, the Lift Persistence framework **will not** be included in the Lift 4.0 release. Before upgrading to Lift 4, you should migrate to a persistence solution not based on the `lift-persistence` set of dependencies.

## What's Changed
* bump sbt version by @lvitaly in https://github.com/lift/framework/pull/2000
* Signing contributors list by @pabloazul in https://github.com/lift/framework/pull/2018
* Lift jetty11 by @pabloazul in https://github.com/lift/framework/pull/2017
* Update docs, config for Java 11 minimum version by @farmdawgnation in https://github.com/lift/framework/pull/2019

## New Contributors
* @pabloazul made their first contribution in https://github.com/lift/framework/pull/2018

**Full Changelog**: https://github.com/lift/framework/compare/3.5.0-release...4.0.0-M1-release

<hr /><em>This discussion was created from the release <a href='https://github.com/lift/framework/releases/tag/4.0.0-M1-release'>4.0.0-M1-release</a>.</em>

**Comment by kenwenzel** [comment]
@farmdawgnation Thank you for the good work. Do you plan a final release of 4.0.0 in the near future?

**Reply by farmdawgnation** [reply]
@kenwenzel Thanks for the ping. I need to get the ball rolling on that, and I appreciate the reminder.
